WebLinking.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name = 'WebLinking'
-  spec.version = '1.1.0'
+  spec.version = '3.0.0'
```

Comment:
Can you please put this back to 1.1.0, we should jump two major releases. I'd also like to handle this in a separate release commit when I've merged the pull request.

Thanks!